### ISSUE TITLE
Optimistic Lock Key breaks with Common tenant

### ISSUE THREAD

**Issue by mstokes-f5** [F5Networks/f5-appsvcs-extension#332]
### Environment
 * Application Services Version: 3.21.0
 * BIG-IP Version: 15.0.1.1

### Summary
When using the optimistic lock key feature with the Common tenant defined, AS3 returns an error (invalid declaration):

```json
{
  "code": 422,
  "errors": [
    "/Common: should NOT have additional properties"
  ],
  "declarationFullId": "",
  "message": "declaration is invalid"
}
```

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "Common": {
        "class": "Tenant",
        "Shared": {
            "class": "Application",
            "template": "shared"
        }
    },
    "class": "ADC",
    "schemaVersion": "3.0.0"
}
```

2. Issue a GET request to `/declare?showHash=true`, and get back:

```json
{
  "Common": {
    "class": "Tenant",
    "Shared": {
      "class": "Application",
      "template": "shared"
    },
    "optimisticLockKey": "dbWTBuWMoHNEDvgOJNEdOO1+iqlHJzsJ5p9aklrthxE="
  },
  "class": "ADC",
  "schemaVersion": "3.0.0",
  "id": "autogen_242459c5-b8cf-4ed7-be3b-052e40b2406f",
  "updateMode": "selective",
  "controls": {
    "archiveTimestamp": "2020-08-31T22:17:23.068Z"
  }
}
```

3. Re-submit earlier declaration with hash (should result in "no change"):

```json
{
    "Common": {
        "class": "Tenant",
        "Shared": {
            "class": "Application",
            "template": "shared"
        },
        "optimisticLockKey": "dbWTBuWMoHNEDvgOJNEdOO1+iqlHJzsJ5p9aklrthxE="
    },
    "class": "ADC",
    "schemaVersion": "3.0.0"
}
```

4. Observe the following error:

```json
{
  "code": 422,
  "errors": [
    "/Common: should NOT have additional properties"
  ],
  "declarationFullId": "",
  "message": "declaration is invalid"
}
```

### Expected Behavior
Expected `optimisticLockKey` to work on Common tenant.

### Actual Behavior
Received an error that the property was invalid.

**Comment by dstokesf5** [comment]
Thank you for reporting this. It has been added to our internal product backlog as AUTOTOOL-1871.

**Comment by dstokesf5** [comment]
This issue has been resolved by the 3.29.0 release.
